Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def device_background_scan_request(self, device_id: Union[list, str]) -> None:
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'BACKGROUND_SCAN',
+            'device_id': device_id,
+            'options': {
+                'toggle': 'ON'
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def device_background_scan_request_stop(self, device_id: Union[list, str]) -> None:
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'BACKGROUND_SCAN',
+            'device_id': device_id,
+            'options': {
+                'toggle': 'OFF'
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def device_policy_update(self, device_id: Union[list, str], policy_id: str) -> None:
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'UPDATE_POLICY',
+            'device_id': device_id,
+            'options': {
+                'policy_id': policy_id
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def list_watchlists_request(self) -> Dict:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists'
+        return self._http_request('GET', suffix_url)
+
+    def get_watchlist_by_id_request(self, watchlist_id: str) -> Dict:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists/{watchlist_id}'
+        return self._http_request('GET', suffix_url)
+
+    def delete_watchlist_request(self, watchlist_id: str) -> None:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists/{watchlist_id}'
+        return self._http_request('DELETE', suffix_url, resp_type='content')
+
+    def watchlist_alert_status_request(self, watchlist_id: str) -> Dict:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists/{watchlist_id}/alert'
+        return self._http_request('GET', suffix_url)
+
+    def enable_watchlist_alert_request(self, watchlist_id: str) -> Dict:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists/{watchlist_id}/alert'
+        return self._http_request('PUT', suffix_url)
+
+    def disable_watchlist_alert_request(self, watchlist_id: str) -> None:
+
+        suffix_url = f'/threathunter/watchlistmgr/v3/orgs/{CB_ORG_KEY}/watchlists/{watchlist_id}/alert'
+        return self._http_request('DELETE', suffix_url, resp_type='content')
+
+    def create_watchlist_request(self, watchlist_name: str, description: str, tags_enabled: bool, alerts_enabled: bool,
```

Comment:
you pass to `report_ids` the ouput for `argToList`, so this will never be a `str`, but only `List`